src/pages/Home/Home.page.js

```diff
@@ -1,10 +1,87 @@
 import React from "react";
-import universalStyles from "./home.module.scss"
+import "./homePage.scss";
+import imgIntro from "./imgHome/img-intro.png";
+import dec from "./imgHome/dec.svg"
+import { Link } from "react-router-dom";
+
+import bicicleta1 from "./imgHome/bicicleta1.png"
+import bicicleta2 from "./imgHome/bicicleta2.png"
+import bicicleta3 from "./imgHome/bicicleta3.png"
+
+import firstIcon from "./imgHome/first.png"
+import secondIcon from "./imgHome/second.png"
+import diferencial from "./imgHome/diferencial.png"
+
 const HomePage = () => {
     return (
-        <div className={universalStyles["container"]}>
-            iuhfgvcxiuhgduifcv
+        <div className="home-container">
+
+            <div className="header-post">
+
+                <img className="header-post-dec" src={dec} alt="dec" />
+                <div className="header-post-text">
+
+                    <div className="header-post-title">bicicletas feitas sob medida<span className="orange-point">.</span></div>
```

Comment:
```suggestion
                    <div className="header-post-title">bicicletas feitas sob medida</div>
```

add dot as pseudo-class

Reply:
done